adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -34,24 +34,43 @@
 from adafruit_esp32spi import adafruit_esp32spi
 import adafruit_esp32spi.adafruit_esp32spi_requests as requests
 
+class WiFiConnType: # pylint: disable=too-few-public-methods
+    """An enum-like class representing the different types of WiFi connections
+    that can be made. The values can be referenced like ``WiFiConnType.normal``.
+    Possible values are
+    - ``ThermocoupleType.normal``
+    - ``ThermocoupleType.enterprise``
+    """
+    # pylint: disable=invalid-name
+    normal = 1
```

Comment:
Make `normal` and `enterprise` be all caps, since these are constants. Then you can take away `# pylint: disable=invalid-name`.

Reply:
Agreed. Fixed as part of one of your other suggestions.